modules/olm-bundle/01_mod.mk

```diff
@@ -106,6 +109,7 @@ olm-publish-$(subst /,-,$1): olm-bundle | $(NEEDS_GH) $(bin_dir)/scratch
 		touch operators/$(olm_project_name)/ci.yaml && \
 		$(if $(and $(findstring redhat-openshift-ecosystem/certified-operators,$1),$(olm_project_id)), \
 			$(YQ) -i '.cert_project_id = "$(olm_project_id)"' operators/$(olm_project_name)/ci.yaml &&) \
+		$(YQ) -i '.updateGraph = "semver-mode"' operators/$(olm_project_name)/ci.yaml && \
```

Comment:
I had a vague memory that you couldn't change this setting, but the documentation says that this is supported, so perhaps  I'm misremembering or the limitation has since been lifted.
 * https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/operator-ci-yaml/#operator-versioning